content/700-optimize/400-recommendations/100-excessive-number-of-rows-returned.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Excessive number of rows returned'
+metaTitle: 'Optimize Recommendations: Excessive number of rows returned'
+metaDescription: "Learn about the recommendation provided by Optimize for excessive number of rows returned."
+tocDepth: 3
+toc: true
+---
+
```

Comment:
Shouldnt there be like an intro paragraph or something? Feels like there is no context